deepmd/pt/model/atomic_model/dp_atomic_model.py

```diff
@@ -223,6 +233,86 @@ def wrapped_sampler():
         if self.fitting_net is not None:
             self.fitting_net.compute_output_stats(wrapped_sampler, stat_file_path)
 
+    def change_out_bias(
```

Comment:
how can other atomic model reuse this method? should it be based in base_atomic_model?

Reply:
Updated in https://github.com/deepmodeling/deepmd-kit/pull/3480/commits/bdf74c916a1490c9be459234339b0411bec43ab2